invenio_app_ils/ui/backoffice/.gitignore

```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
I am not sure we need several `.gitignore` files, personally I add everything on one ignore file on the root of the project.

Reply:
Yes true, but then sometimes would be difficult to ignore some specific files in one of the subprojects especially if they have the same file/folder name.